spot_wrapper/spot_world_objects.py

```diff
@@ -92,3 +97,6 @@ def list_world_objects(
 
         """
         return self._world_objects_client.list_world_objects(object_types, time_start_point)
+
+    def mutate_world_object(self, request: MutateWorldObjectRequest) -> MutateWorldObjectResponse:
```

Comment:
Apologies for being the nittiest of nitpickers, but should this be `mutate_world_objects` to keep with the BD syntax?

Reply:
I was a bit conflicted on this actually because the protobuf message implies that it's singular but the BD syntax is plural (even though their function `mutate_world_objects` only ever deals with mutating a single world object -- it's not possible to pass in multiple objects in the request field)

Reply:
`mutate_world_object(s)`

Reply:
Added this in